### ISSUE TITLE
Non-useful "A new inspection was created." comment on every pull request

### ISSUE THREAD

**Issue by jancborchardt** [scrutinizer-ci/scrutinizer#327]
We use Scrutinizer at ownCloud and like it a lot. :) One thing that annoys me though is the message

> A new inspection was created.

on every single pull request. Nothing happened yet, so notifying me that Scrutinizer started scrutinizing isn’t useful and just an annoying notification.

The messages like

> The inspection completed: No new issues

> The inspection completed: 8 new issues, 48 updated code elements

are awesome, and they are enough. :) No need to have the »New inspection was created« notifications though.

**Comment by schmittjoh** [comment]
Thanks for the suggestion. 

I would suggest to switch the notification strategy to use GitHub's commit status. This is usually not problematic anymore since GitHub now supports multiple different statuses, and not just a single one. 

The comment based notification is only there for legacy reasons, but we plan on removing that eventually.

**Comment by jancborchardt** [comment]
Yeah, we do that for all our app repositories already where we use Travis. The problem with our core however is that we need to use Jenkins (Travis has limits on the open source workers etc.). And Jenkins doesn’t support the multiple-status thingy of Github.

Hence we still need to use the legacy notifications there. And the notification about either success or failure after it has run through is enough.

I can understand if you don’t want to implement since it’s legacy anyway. Just wanted to make sure the issue is known.

**Comment by stof** [comment]
@jancborchardt if Jenkins does not pass a context for its own status, it will just appear as ``Other`` rather than a nice name in the multi-status view on the PR. But this does not prevent you from using multiple statuses (and you could ask maintainers of the jenkins integration to support this properly in future versions btw)

**Comment by jancborchardt** [comment]
Well, the point here is that the notification from _Scrutinizer_ about:

> A new inspection was created.

is not really useful. :) It just fills up my notification dashboard after I created a pull request and closed the tab. It actually has nothing to do with Jenkins.

**Comment by stof** [comment]
@jancborchardt my comment was about you telling you cannot switch to the status-based reporting on Scrutinizer because of Jenkins not supporting multi-status.
When using the status-based notifications, this notification is useful because it marks the build as pending (and gives the link to it to see the progress), which is not the same than not setting any status on github (they display pending statuses in yellow)
